modules/rosa-sts-interactive-mode-reference.adoc

```diff
@@ -51,7 +51,7 @@ You can create a {product-title} cluster with the AWS Security Token Service (ST
 |Enable compute node autoscaling. The autoscaler adjusts the size of the cluster to meet your deployment demands. The default is `No`.
 
 |`Compute nodes`
-|Specify the number of compute nodes to provision into each availability zone. Clusters deployed in a single availability zone require at least 2 nodes. Clusters deployed in multiple zones must have at least 3 nodes. The default is `2`.
+|Specify the number of compute nodes to provision into each availability zone. Clusters deployed in a single availability zone require at least 2 nodes. Clusters deployed in multiple zones must have at least 3 nodes. The default is `2` nodes. The maxmimum number of worker nodes is `180` nodes.
```

Comment:
I don't think you need backticks since you are just writing how many of the thing there are. In the context of how it was originally ("The default is `2`.") that made since because it was really saying "The default value is `2`." I would suggest:

```suggestion
|Specify the number of compute nodes to provision into each availability zone. Clusters deployed in a single availability zone require at least 2 nodes. Clusters deployed in multiple zones must have at least 3 nodes. The maximum number of worker nodes is 180 nodes. The default value is `2`. 
```